plugins/chat/assets/javascripts/discourse/models/chat-channel.js

```diff
@@ -132,16 +129,11 @@ export default class ChatChannel {
     this.tracking = new ChatTrackingState(getOwner(this));
   }
 
-  get unreadThreadCount() {
-    return this.unreadThreadIds.size;
-  }
-
-  get unreadThreadIds() {
-    return this._unreadThreadIds;
-  }
-
-  set unreadThreadIds(unreadThreadIds) {
-    this._unreadThreadIds = new TrackedSet(unreadThreadIds);
+  get unreadThreadsSinceLastViewedCount() {
```

Comment:
You are the english but...

Shouldnt we say `unreadThreadsCountSinceLastViewed` instead?

Also here we pluralize threads but sooner we have `get unreadThreadCount()`, I would tend to use plural here too

Reply:
NAH MATE I'M AUSTRALIAN

Yes your function name is better, will change :)